pkg/cmd/server/server.go

```diff
@@ -55,30 +59,41 @@ const (
 	defaultClientQPS   float32 = 20.0
 	defaultClientBurst int     = 30
 
-	defaultProfilerAddress = "localhost:6060"
+	defaultProfilerAddress         = "localhost:6060"
+	defaultInsecureFlag       bool = true
+	defaultVCConfigFromSecret bool = true
 
 	defaultControllerWorkers = 1
 	// the default TTL for a backup
 	//defaultBackupTTL = 30 * 24 * time.Hour
 )
 
 type serverConfig struct {
-	metricsAddress                        									string
-	clientQPS                                                               float32
-	clientBurst                                                             int
-	profilerAddress                                                         string
-	formatFlag                                                              *logging.FormatFlag
+	metricsAddress     string
+	clientQPS          float32
+	clientBurst        int
+	profilerAddress    string
+	formatFlag         *logging.FormatFlag
+	vCenter            string
+	port               string
+	user               string
+	clusterId          string
+	insecureFlag       bool
+	vcConfigFromSecret bool
 }
 
 func NewCommand(f client.Factory) *cobra.Command {
 	var (
-		logLevelFlag            = logging.LogLevelFlag(logrus.InfoLevel)
-		config                  = serverConfig{
-			metricsAddress:                    defaultMetricsAddress,
-			clientQPS:                         defaultClientQPS,
-			clientBurst:                       defaultClientBurst,
-			profilerAddress:                   defaultProfilerAddress,
-			formatFlag:                        logging.NewFormatFlag(),
+		logLevelFlag = logging.LogLevelFlag(logrus.InfoLevel)
+		config       = serverConfig{
+			metricsAddress:     defaultMetricsAddress,
+			clientQPS:          defaultClientQPS,
+			clientBurst:        defaultClientBurst,
+			profilerAddress:    defaultProfilerAddress,
+			formatFlag:         logging.NewFormatFlag(),
+			port:               utils.DefaultVCenterPort,
```

Comment:
I think for uniformity sake we can have the default vc port in the consts of this file instead of fetching it from utils, looks cleaner imo.

Reply:
I think its good to avoid duplicate code, so might be OK to reference the utils value. We use the utils.DefaultVCenterPort value incase the port is not specified in the secret file. This is a similar usecase.